letsencrypt/models/letsencrypt.py

```diff
@@ -8,26 +8,26 @@
 import subprocess
 import tempfile
 from openerp import _, api, models, exceptions
-from openerp.tools.config import _get_default_datadir
+from openerp.tools import config
 
 
 DEFAULT_KEY_LENGTH = 4096
 _logger = logging.getLogger(__name__)
 
 
+def get_data_dir():
```

Comment:
those are model functions on purpose to make changing this by inheritance simpler. Any technical reason to change this?